node_modules/oae-core/termsandconditions/js/termsandconditions.js

```diff
@@ -21,6 +21,27 @@ define(['jquery', 'oae.core'], function($, oae) {
         var $rootel = $('#' + uid);
 
         /**
+         * Accept the invitation in the URL state, if any
+         */
+        var acceptInvitation = function() {
+            var invitationToken = oae.api.util.url().param('invitationToken');
```

Comment:
Does this mean that invitation won't be accepted in the following scenario:

1. The user accepts their invitation in a way that doesn't supply a confirmed email address
2. The user enters their email address
3. The user clicks the verify link in the verify email
4. The user is now shown the T&C modal, but the `invitationToken` is no longer present in the URL because they followed the verify link